tiledb/sm/serialization/array_schema.cc

```diff
@@ -29,8 +29,12 @@
  *
  * This file defines serialization functions for ArraySchema.
  */
+#ifdef TILEDB_SERIALIZATION
+#include <capnp/compat/json.h>
+#include <capnp/serialize.h>
+#include "tiledb/sm/serialization/capnp_utils.h"
```

Comment:
again ? similar to above, capnp_utils.h inclusion not previously subject to TILEDB_SERIALIZATION, does any build configuration not enabling capnp support still require it?